build-system/global-configs/canary-config.json

```diff
@@ -24,5 +24,4 @@
   "a4a-measure-get-ad-urls": 1,
   "ad-loader-v2": 1,
   "3p-use-ampcontext": 0,
-  "amp-ad-exit": 1
```

Comment:
do the config change in a separate PR. otherwise it will fail our presubmit check